velox/exec/Unnest.cpp

```diff
@@ -64,18 +67,26 @@ RowVectorPtr Unnest::getOutput() {
   }
 
   auto size = input_->size();
+  inputRows_.resize(size);
 
-  // Build repeated indices to apply to "replicated" columns.
   const auto& unnestVector = input_->childAt(unnestChannel_);
-
-  inputRows_.resize(size);
   unnestDecoded_.decode(*unnestVector, inputRows_);
-
   auto unnestIndices = unnestDecoded_.indices();
 
-  auto unnestBase = unnestDecoded_.base()->as<ArrayVector>();
-  auto rawSizes = unnestBase->rawSizes();
-  auto rawOffsets = unnestBase->rawOffsets();
+  const ArrayVector* unnestDecodedBaseArrayVector;
```

Comment:
These names are very long. Consider shortening to baseArray and baseMap or unnestArrayBase and unnestMapBase.